### PR TITLE
CAKE-5064 Allow rejection of non-IAB vendors

### PR DESCRIPTION
**Pull request by mklucsarits**
This covers the UI for rejecting tracking for non-IAB vendors.

### CODE REVIEW

File: src/components/OtherPartners.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        onToggle(isOn);
+        tracker.trackOtherPartnersToggleClick();
+    }
+
+    renderVendors() {
+        const { content } = this.props;
+
+        const toRender = NON_IAB_VENDORS.map((vendor) => {
+            return (
+            <div className={vendorListStyles.vendor}>
```
Comment (JoshuaRogan) [inline src/components/OtherPartners.js]: Do you need to add `keys` to these components?

Comment (mklucsarits) [reply src/components/OtherPartners.js]: Probably, although there's no apparent number to use.

Comment (JoshuaRogan) [reply src/components/OtherPartners.js]: It can be the name of the vendor. It doesn't need to be a number.

Comment (mklucsarits) [reply src/components/OtherPartners.js]: I just added keys for every loop :)

Comment (JoshuaRogan) [reply src/components/OtherPartners.js]: 🍠 I wonder if this has anything to do with requiring the `forceUpdate`. Maybe we aren't doing that elsewhere?

**Pull request merged**